lms/views/email.py

```diff
@@ -11,23 +11,20 @@
 LOG = logging.getLogger(__name__)
 
 
-@view_config(
+@forbidden_view_config(
+    route_name="email.preferences",
+    request_method="GET",
+    request_param="token",
+    renderer="lms:templates/email/expired_link.html.jinja2",
+)
```

Comment:
Some duplication in the `@forbidden_view_config`'s here but AFAIK this is the only way to get one view callable to apply to two different routes. We could write our own decorator that wraps Pyramid's `@forbidden_view_config` decorator but that would be OTT to save one instance of duplication.